SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security
+
+This document specifies the CVE (Common Vulnerabilities and Exposure) process
+for the SwiftNIO project.
+
+## Disclosures
+
+### Private Disclosure Process
+
+The SwiftNIO core team asks that known and suspected vulnerabilities be
+privately and responsibly disclosed by emailing TODO with the
+[details usually included with bug reports][issue-template]. **Do not file a
+public issue.**
+
+#### When to report a vulnerability
+
+* You think you have discovered a potential security vulnerability in SwiftNIO
+  or any of the SwiftNIO projects.
+* You are unsure how a vulnerability affects SwiftNIO or any of the SwiftNIO
+  projects.
+
+#### What happens next?
+
+* A member of the team will acknowledge receipt of the report within X
+  working days. This may include a request for additional information about
+  reproducing the vulnerability.
+* We will inform the Swift Server Work Group ([SSWG][sswg]) of the vulnerability
+  within 10 days of the report.
+* Once we have identified a fix we may ask you to validate it. We aim to do this
```

Comment:
X=30?

We should add that sometimes it's a coordinated disclosure process (example the HTTP/2 vulns) where the industry coordinates in which case we can't guarantee much.

Reply:
Ah, that's a really good point.